ext/RMagick/rmfill.c

```diff
@@ -197,7 +197,7 @@ vertical_fill(
     // it to each actual row.
     master = ALLOC_N(PixelPacket, image->columns);
 
-    for (x = 0; x < image->columns; x++)
+    for (x = 0; x < (ssize_t) image->columns; x++)
```

Comment:
unnecessary type cast.
```suggestion
    for (x = 0; x < image->columns; x++)
```